src/python/google_cloud_utils/gsutil.py

```diff
@@ -137,7 +143,7 @@ def rsync(self,
   def download_file(self, gcs_url, file_path, timeout=None):
     """Download a file from GCS."""
     command = ['cp', _filter_path(gcs_url), file_path]
-    result = self.gsutil_runner.run_and_wait(command, timeout=timeout)
+    result = self.run_gsutil(command, timeout=timeout)
```

Comment:
We should probably have something in run_gsutil that converts command into a list. in future, if we add quiet flag, it will exception out.

Reply:
Could you please clarify this comment? `command` is already a list?

Reply:
ah sorry didn't know that, ignore.